app/assets/stylesheets/styles.css

```diff
@@ -44,13 +44,9 @@
   }
 }
 
-html,
-body {
```

Comment:
Having `body` here was causing an issue on the home page with the logged-in header having `position: sticky`. The header would scroll up with the hero. Removing `body` fixes that.